### ISSUE TITLE
i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220 (attempt to speed up NOCOMP mode)

### ISSUE THREAD

**Issue by rosiealice** [NorESMhub/NorESM3_land_only_dev_simulations#6]
**Purpose:**
To test whether the model runs faster than n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c if we modify the path fusion parameters . The parameter file differences look like this: 

<  fates_cohort_age_fusion_tol = 0.16 ;
---
>  fates_cohort_age_fusion_tol = 0.08 ;
1727c1727
<  fates_cohort_size_fusion_tol = 0.16 ;
---
>  fates_cohort_size_fusion_tol = 0.08 ;
1821c1821
<  fates_patch_fusion_tol = 0.30 ;
---
>  fates_patch_fusion_tol = 0.05 ;

__
**Description:**

To get the code
> git clone https://github.com/NorESMhub/NorESM.git
> cd NorESM
> git checkout noresm2_5_alpha08d
> ./bin/git-fleximod update

- NOTE: a memory leak is present in the run. 
See https://github.com/NorESMhub/NorESM/issues/550

- **create_newcase commnad**
```
./create_newcase --case i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220 --compset 1850_DATM%QIA_CLM60%FATES_SICE_SOCN_MOSART_SGLC_SWAV --res ne30pg3_tn14 --project NN9188K --run-unsupported --mach betzy -pecount L
- **user_nl_clm**


fates_paramfile = '/cluster/work/users/rosief/git/calibration_parameter_files/NOCOMP_calib/fates_params_alpha09_fast2.nc'

use_fates_nocomp=TRUE
use_fates_fixed_biogeog=TRUE
hist_fexcl1 =
	      'FATES_VEGC_PF',
              'FATES_STOREC_PF', 'FATES_RECRUITMENT_PF', 'FATES_MORTALITY_PF', 'FATES_PATCHAREA_AP', 'FATES_LAI_AP',
              'FATES_CANOPYAREA_AP', 'FATES_NESTEROV_INDEX', 'FATES_IGNITIONS', 'FATES_FDI', 'FATES_ROS', 'FATES_EFFECT_WSPEED',
              'FATES_FUELCONSUMED', 'FATES_FIRE_INTENSITY', 'FATES_FIRE_INTENSITY_BURNFRAC', 'FATES_BURNFRAC', 'FATES_FUEL_MEF',
              'FATES_FUEL_BULKD', 'FATES_FUEL_EFF_MOIST', 'FATES_FUEL_SAV', 'FATES_FUEL_AMOUNT', 'FATES_FRAGMENTATION_SCALER_SL',
              'FATES_FUEL_MOISTURE_FC', 'FATES_FUEL_AMOUNT_FC', 'FATES_BURNFRAC_AP', 'FATES_FIRE_INTENSITY_BURNFRAC_AP',
              'FATES_FUEL_AMOUNT_AP', 'FATES_FUEL_BURNT_BURNFRAC_FC',
	      'FATES_LITTER_IN_EL', 'FATES_LITTER_OUT_EL', 'FATES_SEED_BANK_EL', 'FATES_SEEDS_IN_LOCAL_EL',
              'FATES_SEEDS_IN_EXTERN_EL', 'FATES_SEED_GERM_EL', 'FATES_SEED_DECAY_EL', 'FATES_STOREC', 'FATES_VEGC',
              'FATES_SAPWOODC', 'FATES_FROOTC', 'FATES_REPROC', 'FATES_STRUCTC', 'FATES_NONSTRUCTC', 
              'FATES_VEGC_ABOVEGROUND', 'FATES_CANOPY_VEGC', 'FATES_USTORY_VEGC', 'FATES_PRIMARY_PATCHFUSION_ERR',
              'FATES_DISTURBANCE_RATE_FIRE', 'FATES_DISTURBANCE_RATE_LOGGING', 'FATES_DISTURBANCE_RATE_TREEFALL',
              'FATES_HARVEST_WOODPROD_C_FLUX', 'FATES_GPP_CANOPY', 'FATES_AUTORESP_CANOPY',
              'FATES_GPP_USTORY', 'FATES_AUTORESP_USTORY', 'FATES_CROWNAREA_CL', 'FATES_DEMOTION_CARBONFLUX',
              'FATES_PROMOTION_CARBONFLUX', 'FATES_MORTALITY_CFLUX_CANOPY', 'FATES_MORTALITY_CFLUX_USTORY', 
              'FATES_DDBH_CANOPY_SZ', 'FATES_DDBH_USTORY_SZ', 'FATES_BASALAREA_SZ',
              'FATES_VEGC_ABOVEGROUND_SZ', 'FATES_LAI_CANOPY_SZ', 'FATES_MORTALITY_CANOPY_SZ', 'FATES_NPLANT_USTORY_SZ',
              'FATES_LAI_USTORY_SZ', 'FATES_NPLANT_SZ', 'FATES_NPLANT_AC', 'FATES_MORTALITY_BACKGROUND_SZ',
              'FATES_MORTALITY_HYDRAULIC_SZ', 'FATES_MORTALITY_CSTARV_SZ', 'FATES_MORTALITY_IMPACT_SZ',
              'FATES_MORTALITY_FIRE_SZ', 'FATES_MORTALITY_TERMINATION_SZ', 'FATES_MORTALITY_LOGGING_SZ',
              'FATES_MORTALITY_FREEZING_SZ', 'FATES_MORTALITY_SENESCENCE_SZ', 'FATES_MORTALITY_AGESCEN_SZ',
              'FATES_MORTALITY_AGESCEN_AC', 'FATES_MORTALITY_USTORY_SZ', 'FATES_FROOTMAINTAR', 'FATES_CROOTMAINTAR',
              'FATES_LSTEMMAINTAR', 'FATES_NEP', 'FATES_HET_RESP', 'FATES_FIRE_CLOSS', 'FATES_FIRE_FLUX_EL',
              'FATES_LITTER_AG_FINE_EL', 'FATES_LITTER_BG_FINE_EL', 'FATES_LITTER_BG_CWD_EL', 'FATES_LITTER_AG_CWD_EL',
              'FATES_LITTER_CWD_ELDC',
'FATES_NPP_PF','FATES_MORTALITY_LOGGING_SE_SZ',
'FATES_MORTALITY_HYDRAULIC_SE_SZ','FATES_MORTALITY_FREEZING_SE_SZ',
'FATES_MORTALITY_CSTARV_SE_SZ','FATES_MORTALITY_CANOPY_SE_SZ',
'FATES_MORTALITY_CANOPY_SE_SZ','FATES_MORTALITY_BACKGROUND_SE_SZ',
'FATES_MORTALITY_BACKGROUND_SE_SZ','FATES_MORTALITY_AGESCEN_SE_SZ',
'FATES_VEGC_SE_PF','FATES_RECRUITMENT_CFLUX_PF','FATES_NPP_SE_PF',
'FATES_NPP_SE_PF','FATES_NPLANT_SEC_PF','FATES_NOCOMP_PATCHAREA_PF',
'FATES_NOCOMP_NPATCHES_PF','FATES_NOCOMP_BURNEDAREA_PF',
'FATES_MORT_CSTARV_CONT_CFLUX_PF','FATES_MORTALITY_HYDRO_CFLUX_PF',
'FATES_MORTALITY_FIRE_CFLUX_PF','FATES_MORTALITY_CSTARV_CFLUX_PF',
'FATES_MORTALITY_CFLUX_PF','FATES_MEANSMP_DROUGHTPHEN_PF',
'FATES_MEANLIQVOL_DROUGHTPHEN_PF','FATES_LEAFC_PF',
'FATES_L2FR_USTORY_REC_PF','FATES_L2FR_CANOPY_REC_PF',
'FATES_GPP_SE_PF','FATES_ELONG_FACTOR_PF','FATES_DROUGHT_STATUS_PF',
'FATES_DAYSINCE_DROUGHTLEAFON_PF','FATES_DAYSINCE_DROUGHTLEAFOFF_PF',
'FATES_CANOPYCROWNAREA_PF','FATES_TRANSITION_MATRIX_LULU',
'FATES_DISTURBANCE_RATE_MATRIX_LULU','FATES_PATCHAREA_LU',;
'
```
- **xmlchange commands**
```
./xmlchange STOP_OPTION=nyears
./xmlchange STOP_N=2
./xmlchange RESUBNIT=20


```
___
**Compset long name**
1850_DATM%QIA_CLM60%FATES_SICE_SOCN_MOSART_SGLC_SWAV
___
**Model grid long name**
`a%ne30np4.pg3_l%ne30pg4.pg3_oi%tnx1v4_r%r05_w%null_z%null_g%gris4_m%tnx1v4`

**Case directory:**
- Locally on betzy: 
   `/cluster/work/users/rosief/git/noresm2_5_alpha08d/cime/scripts/ i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220 

- On github:
tbd
___
**Code version on github (either NorESMhub or user's fork):**
- On github:  https://github.com/NorESMhub/NorESM/releases/tag/noresm2_5_alpha08d
- https://github.com/username/noresm/tree/<code_version>
___
**Diagnostics:**
- Diags (if available):
https://ns9560k.web.sigma2.no/datalake/diagnostics/noresm/rosief/ i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220 

Comparison plots here:
https://ns9560k.web.sigma2.no/datalake/diagnostics/noresm/rosief/i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220/compare/n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c/ANN/

TL:DR This run has a low LAI and lots of patches... 
___
**Output:**
- Raw output on nird (if still available): 
/datalake/NS9560K/noresm3/cases/ i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220 /lnd/hist/ 
___
**Contacts:**
@rosiealice 
___
**Extra details:**

**Comment by rosiealice** [comment]
Looking at the timing files in the case/timing directory and doing 
grep TOT cesm_timing.*
gives

This run 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047382.241220-130603:    TOT Run Time:    1083.999 seconds        2.970 seconds/mday        79.70 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047394.241220-132754:    TOT Run Time:    1149.034 seconds        3.148 seconds/mday        75.19 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047401.241220-135239:    TOT Run Time:    1246.264 seconds        3.414 seconds/mday        69.33 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047422.241220-141800:    TOT Run Time:    1381.386 seconds        3.785 seconds/mday        62.55 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047431.241220-144543:    TOT Run Time:    1499.938 seconds        4.109 seconds/mday        57.60 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047449.241220-154301:    TOT Run Time:    1456.324 seconds        3.990 seconds/mday        59.33 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047464.241220-161240:    TOT Run Time:    1535.003 seconds        4.205 seconds/mday        56.29 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047477.241220-164257:    TOT Run Time:    1535.581 seconds        4.207 seconds/mday        56.27 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047484.241220-171329:    TOT Run Time:    1578.883 seconds        4.326 seconds/mday        54.72 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047505.241220-174430:    TOT Run Time:    1605.117 seconds        4.398 seconds/mday        53.83 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047517.241220-181546:    TOT Run Time:    1713.572 seconds        4.695 seconds/mday        50.42 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047521.241220-184914:    TOT Run Time:    1691.808 seconds        4.635 seconds/mday        51.07 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047526.241220-192201:    TOT Run Time:    1699.563 seconds        4.656 seconds/mday        50.84 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047533.241220-195519:    TOT Run Time:    1796.077 seconds        4.921 seconds/mday        48.10 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047547.241220-203002:    TOT Run Time:    1773.264 seconds        4.858 seconds/mday        48.72 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047557.241220-210521:    TOT Run Time:    1813.933 seconds        4.970 seconds/mday        47.63 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047560.241220-214040:    TOT Run Time:    1823.041 seconds        4.995 seconds/mday        47.39 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047566.241220-221604:    TOT Run Time:    1859.658 seconds        5.095 seconds/mday        46.46 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047576.241220-225149:    TOT Run Time:    1857.839 seconds        5.090 seconds/mday        46.51 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047589.241220-232854:    TOT Run Time:    1864.410 seconds        5.108 seconds/mday        46.34 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047594.241221-000524:    TOT Run Time:    1891.205 seconds        5.181 seconds/mday        45.69 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047598.241221-004225:    TOT Run Time:    1886.594 seconds        5.169 seconds/mday        45.80 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047604.241221-011833:    TOT Run Time:    1884.378 seconds        5.163 seconds/mday        45.85 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047606.241221-015454:    TOT Run Time:    1912.628 seconds        5.240 seconds/mday        45.17 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047608.241221-023145:    TOT Run Time:    1908.764 seconds        5.229 seconds/mday        45.26 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047610.241221-030903:    TOT Run Time:    1941.229 seconds        5.318 seconds/mday        44.51 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047613.241221-034641:    TOT Run Time:    1916.984 seconds        5.252 seconds/mday        45.07 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047678.241221-042402:    TOT Run Time:    1933.118 seconds        5.296 seconds/mday        44.69 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047784.241221-050126:    TOT Run Time:    1928.364 seconds        5.283 seconds/mday        44.80 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047786.241221-053902:    TOT Run Time:    2002.600 seconds        5.487 seconds/mday        43.14 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047792.241221-061731:    TOT Run Time:    1965.860 seconds        5.386 seconds/mday        43.95 myears/wday 
cesm_timing.i1850.FATES-NOCOMP-FAST.ne30_tn14.alpha08d.20241220.1047796.241221-065528:    TOT Run Time:    1940.030 seconds        5.315 seconds/mday        44.54 myears/wday 


default run
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046252.241218-220049:    TOT Run Time:    2140.049 seconds        2.932 seconds/mday        80.75 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046278.241218-224016:    TOT Run Time:    3056.447 seconds        4.187 seconds/mday        56.54 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046663.241219-110327:    TOT Run Time:    1846.230 seconds        5.058 seconds/mday        46.80 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046688.241219-113855:    TOT Run Time:    1945.940 seconds        5.331 seconds/mday        44.40 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046699.241219-121700:    TOT Run Time:    1978.897 seconds        5.422 seconds/mday        43.66 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046737.241219-125506:    TOT Run Time:    2042.856 seconds        5.597 seconds/mday        42.29 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046771.241219-133334:    TOT Run Time:    2082.173 seconds        5.705 seconds/mday        41.50 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046816.241219-141258:    TOT Run Time:    2161.182 seconds        5.921 seconds/mday        39.98 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046847.241219-145435:    TOT Run Time:    2169.123 seconds        5.943 seconds/mday        39.83 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046875.241219-153526:    TOT Run Time:    2221.893 seconds        6.087 seconds/mday        38.89 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046919.241219-161728:    TOT Run Time:    2300.223 seconds        6.302 seconds/mday        37.56 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1046953.241219-170026:    TOT Run Time:    2384.217 seconds        6.532 seconds/mday        36.24 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047013.241219-174523:    TOT Run Time:    2420.468 seconds        6.631 seconds/mday        35.70 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047030.241219-183041:    TOT Run Time:    2449.305 seconds        6.710 seconds/mday        35.28 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047047.241219-191637:    TOT Run Time:    2458.005 seconds        6.734 seconds/mday        35.15 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047065.241219-200245:    TOT Run Time:    2477.614 seconds        6.788 seconds/mday        34.87 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047070.241219-204857:    TOT Run Time:    2480.634 seconds        6.796 seconds/mday        34.83 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047078.241219-213512:    TOT Run Time:    2496.019 seconds        6.838 seconds/mday        34.62 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047082.241219-222120:    TOT Run Time:    2555.464 seconds        7.001 seconds/mday        33.81 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047096.241219-230855:    TOT Run Time:    2517.258 seconds        6.897 seconds/mday        34.32 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047105.241219-235556:    TOT Run Time:    2543.590 seconds        6.969 seconds/mday        33.97 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047111.241220-004359:    TOT Run Time:    2583.927 seconds        7.079 seconds/mday        33.44 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047219.241220-013323:    TOT Run Time:    2577.086 seconds        7.061 seconds/mday        33.53 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047226.241220-022127:    TOT Run Time:    2619.419 seconds        7.176 seconds/mday        32.98 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047230.241220-031018:    TOT Run Time:    2625.568 seconds        7.193 seconds/mday        32.91 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047235.241220-035933:    TOT Run Time:    2593.807 seconds        7.106 seconds/mday        33.31 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047237.241220-044845:    TOT Run Time:    2581.280 seconds        7.072 seconds/mday        33.47 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047239.241220-053744:    TOT Run Time:    2616.016 seconds        7.167 seconds/mday        33.03 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047249.241220-062636:    TOT Run Time:    2614.475 seconds        7.163 seconds/mday        33.05 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047251.241220-071543:    TOT Run Time:    2621.011 seconds        7.181 seconds/mday        32.96 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047253.241220-080539:    TOT Run Time:    2632.271 seconds        7.212 seconds/mday        32.82 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047267.241220-085455:    TOT Run Time:    2659.431 seconds        7.286 seconds/mday        32.49 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047298.241220-094445:    TOT Run Time:    2655.883 seconds        7.276 seconds/mday        32.53 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047322.241220-103419:    TOT Run Time:    2670.603 seconds        7.317 seconds/mday        32.35 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047345.241220-112429:    TOT Run Time:    2638.386 seconds        7.228 seconds/mday        32.75 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047360.241220-121441:    TOT Run Time:    2668.904 seconds        7.312 seconds/mday        32.37 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047380.241220-130423:    TOT Run Time:    2674.335 seconds        7.327 seconds/mday        32.31 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047403.241220-135415:    TOT Run Time:    2653.088 seconds        7.269 seconds/mday        32.57 myears/wday 
cesm_timing.n1850.FATES-NOCOMP.ne30_tn14.alpha08d.20241218c.1047429.241220-144340:    TOT Run Time:    2667.863 seconds        7.309 seconds/mday        32.39 myears/wday 

So this appears to be significantly faster (44 vs 32 years per day at around 40 years)

**Comment by rosiealice** [comment]
The differences in LAI do not show up as especially important 

![Image](https://github.com/user-attachments/assets/3cbae27a-2236-4343-9c87-1f695ffe4d39)

**Comment by rosiealice** [comment]
There appear to be a lot of grassland patches. I wonder if the logic of the relative differences needs modification.

![Image](https://github.com/user-attachments/assets/1f69e823-6d13-4632-a9d3-fe43da98cb09)

![Image](https://github.com/user-attachments/assets/383c65a9-989c-4015-bfe5-9d5bc9e944a1)
